node-operators/block-producers/onboarding/run-a-block-producer/block-producer-systemd.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Run a Tanssi Block Producer
+description: Learn how to set up and run block producers (collators) for Tanssi Appchains using Systemd, allowing you to participate in the protocol and earn rewards.
+---
+
+# Run a Block Producer Node Using Systemd
+
+## Introduction {: #introduction }
+
+--8<-- 'text/node-operators/block-producers/run-a-block-producer/intro.md'
+
+In this guide, you'll learn how to spin up a Tanssi block producer to be part of the shared pool of sequencers using the latest stable binary file release and managing the service with [Systemd](https://systemd.io/){target=\_blank} on Linux systems.
+
+The article follows the good practice of running the service with its own non-root account and granting that account write access to a specific directory. However, you can adapt this article's steps and instructions to your infrastructure configuration, preferences, and security policies.
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+To get started, you'll need access to a computer running an Ubuntu Linux OS and root privileges. You will also need:
+
+- **Node binary file** - the instructions in this guide execute the [latest](https://github.com/moondance-labs/tanssi/releases/latest){target=\_blank} official stable `tanssi-node` release. However, you can build your own file compiling the [source code](https://github.com/moondance-labs/tanssi){target=\_blank}
+
+- **Tanssi orchestrator specifications file** - the Tanssi orchestrator specifications file is needed to run the block producer. You can download it from this [public GitHub repository](https://github.com/papermoonio/external-files/blob/main/Tanssi/Dancebox/){target=\_blank}
+
+- **Relay chain specifications file** - the relay chain specification file can be downloaded from this [public GitHub repository](https://github.com/papermoonio/external-files/blob/main/Moonbeam/Moonbase-Alpha/){target=\_blank}
+
+## Download the Latest Release {: #download-latest-release }
+
+To get started, download and make executable the latest binary release by running the following command:
+
+```bash
+wget https://github.com/moondance-labs/tanssi/releases/latest/download/tanssi-node && \
+chmod +x ./tanssi-node
+```
+
+--8<-- 'text/node-operators/optimized-binaries-note.md'
```

Comment:
With the modified text, this should be above the commands so reader gets it first before downloading the binary